### ISSUE TITLE
Step count on elevation decrease

### ISSUE THREAD

**Issue by rorybee** [grmpl/StepandHeightcounter#18]
Step counts are incrementing on climbing down (elevation decrease) on Android 11. Is that expected? It might be better to separate step counts for elevation increase/decrease?

**Comment by grmpl** [comment]
Oh! I think you have the wrong interpretation of steps: The app is counting **foot**steps not staircase-steps. So every walking step is counted and it doesn't matter if it's up, down or horizontally. So incrementation of steps on climbing down is correct.
I do have to clarify this in the documentation.

**Comment by rorybee** [comment]
Yes, I understand it's technically counting steps, which does correspond to stairs practically. So it might be a good feature to log steps separately if elevation is increasing or decreasing .

**Comment by grmpl** [comment]
You are right: It would be interesting how many steps are done upwards, downwards or horizontally each. I will think about this feature.
